bitbots_docs/docs/manual/hardware/wolfgang_core.rst

```diff
@@ -0,0 +1,314 @@
++--------+--------+---------------------------------------+--------+---------+---------+-------------+
+
+.. _DXL:
+
+DXL
+---
+
+**id**: Can be a value between 1 and 252. it is used to talk to the device over the Dynamixel bus.
+
+**baud**: Can be a value between 0 and 7
+
++-------+---------+--------+
+| value | baud    | Tested |
++=======+=========+========+
+| 0     | 9,600   | no     |
++-------+---------+--------+
+| 1     | 57,600  | no     |
++-------+---------+--------+
+| 2     | 115,200 | no     |
++-------+---------+--------+
+| 3     | 1M      | no     |
++-------+---------+--------+
+| 4     | 2M      | yes    |
++-------+---------+--------+
+| 5     | 3M      | no     |
++-------+---------+--------+
+| 6     | 4M      | yes    |
++-------+---------+--------+
+| 7     | 4.5M    | no     |
++-------+---------+--------+
+
+We are reasonably certain that the other baud rates work as well since the Teensy supports them.
+
+
+.. _LEDs:
+
+LEDs
+----
+
+**led{0,1,2}**: Byte order: RGB, 4th byte is ignored but reserved.
+
+
+.. _Power Control:
+
+Power Control
+-------------
+
+**power_control**: Used to turn on and off the power (0 off, 1 on). Will be overwritten by manual switch if toggled.
+
+**manual_power_on**: Indicates whether the manual power switch is on. Requires :ref:`Bodge Switch`
+
+.. _Voltage Sensing:
+
+Voltage Sensing
+---------------
+
+Voltages are scaled down using a voltage divider to be read by the microcontroller.
+Multiplying by the given scale factor returns the actual voltage on the voltage rail.
+The scale factor is given as the conversion factor from analog reading to voltage multiplied by the factor of the voltage divider.
+The factor of the voltage divider is given as (top_resistor/(top_resistor+bottom_resistor)).
+It is recommended to use ±0.1% or ±1% resistors for the voltage dividers.
+
+**VEXT**: Raw reading of the external power supply voltage. Scale factor: (3.3 / 1024) * (2.0/(2.0+10.0))
+
+**VCC**: Raw reading of the main voltage rail. Scale factor: (3.3 / 1024) * (2.0/(2.0+10.0))
+
+**VDXL**: Raw reading of the voltage applied to the Dynamixel bus. Scale factor: (3.3 / 1024) * (2.0/(2.0+10.0))
+
+**VBAT_{0..5}**: Raw reading of the voltage between ground and cell {0..5}.
+
+**VBAT_0**: Scale factor: (3.3 / 1024) * (3.3/(1.2+3.3))
+
+**VBAT_1**: Scale factor: (3.3 / 1024) * (3.6/(6.2+3.6))
+
+**VBAT_2**: Scale factor: (3.3 / 1024) * (2.2/(6.8+2.2))
+
+**VBAT_3**: Scale factor: (3.3 / 1024) * (3.6/(16.0+3.6))
+
+**VBAT_4**: Scale factor: (3.3 / 1024) * (6.2/(36.0+6.2))
+
+**VBAT_5**: Scale factor: (3.3 / 1024) * (1.8/(13.0+1.8))
+
+
+.. _Current Sensing:
+
+Current Sensing
+---------------
+
+Current is sensed using a Hall effect sensor (ACS712ELCTR-30A-T to be exact).
+It has to be scaled by the conversion factor from analog reading to voltage multiplied by the amperes per volt to get the actual current.
```

Comment:
amperes per volt?

Reply:
yes, the voltage the sensor outputs goes up/down by 0.066 V per ampere that is flowing through it